database/migrations/2017_05_20_120000_create_posts_table.php

```diff
@@ -32,6 +32,8 @@ public function up()
             $table->dateTime('published_at'); // Allows blog posts to be published at a later date.
             $table->softDeletes();
             $table->timestamps();
+
+            $table->unique(['slug', 'series_id']);
```

Comment:
Duplicate post slugs are OK, provided they are nested within different series.

Reply:
Posts (#5) are the exception where I would rather see them truly unique on slugs with what I want to do allowing them to have routes at `/blog/{post}` and be redirected to the actual path of the Post if it belongs to a Series (#4).